sklearn/preprocessing/tests/test_polynomial.py

```diff
@@ -552,6 +552,33 @@ def test_polynomial_features_csr_X(deg, include_bias, interaction_only, dtype):
     assert_array_almost_equal(Xt_csr.A, Xt_dense)
 
 
+@pytest.mark.parametrize("columns", [1, 2, 3, 1000])
+def test_polynomial_features_csr_wide(columns):
+    x = sparse.csr_matrix(([1], ([0], [columns])))
+
+    est = PolynomialFeatures(3)
+    # this causes a crash with values of `columns` which are too large
```

Comment:
Could you be more precise? For example `# regression test for gh-issue #16803`. In addition,  I guess this does not crash for `columns in [1, 2, 3]`.

Reply:
If the test does not fail while #16803 is not (yet) fixed, this test does not really test anything. In that case, please remove it - or tell me if I misunderstood something.

Reply:
removed. I had basically this same discussion earlier and I justified it as a sort of performance regression test 
https://github.com/scikit-learn/scikit-learn/pull/19734#discussion_r598243605
but 🤷